src/pages/productpage/allproducts/allproducts.css

```diff
@@ -75,18 +75,27 @@
     all:unset;
 }
 .add-to-wishlist i{
-    color: #000000;
+    color: red;
 }
 .add-to-wishlist i:hover {
     color: #e31b23;
```

Comment:
You can use a variable, if you are re-using this color somewhere else

Reply:
will work on this saran👍